objectherkenning_openbare_ruimte/frame_extraction/frame_extraction.py

```diff
@@ -0,0 +1,105 @@
+import argparse
+import os
+import pathlib
+import sys
+from datetime import datetime
+
+sys.path.append(os.getcwd())
+
+from objectherkenning_openbare_ruimte.settings.settings import (  # noqa: E402
+    ObjectherkenningOpenbareRuimteSettings,
+)
+
+config_path = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "..", "config.yml")
+)
+
+ObjectherkenningOpenbareRuimteSettings.set_from_yaml(config_path)
+settings = ObjectherkenningOpenbareRuimteSettings.get_settings()
+
+
+def main(input_folder: str, output_folder: str):
+    """
+    Recursively iterates through all MP4 files in the input_folder and its subfolders, extracts frames at a given framerate, and applies distortion correction.
+
+    All frames are extracted to the same folder, with their names reflecting the name of the source video file. In a separate folder log files will be written, one general with the executed ffmpeg commands, and individual log files for each ffmpeg operation.
+
+    Usage:
+    python frame_extraction.py --input_folder <INPUT> --output_folder <OUTPUT>
+
+    Note:
+    ffmpeg >= v4.4.3 is required
+
+    Parameters
+    ----------
+    input_folder: str
+        Folder containing MP4 files to be processed.
+    output_folder: str
+        Folder to which extracted frames and log files will be written.
+    """
+    fe_settings = settings["frame_extraction"]
```

Comment:
What does fe and dc stands for? Is it possible to have a longer name that adds clarity?